docs/user_guide/further_reading.rst

```diff
@@ -0,0 +1,24 @@
+.. _further_reading:
+
+Further Reading
+===============
+
+There are a number of sources of further information about fairness in machine learning.
+Here we list which concentrate more on the broader societal impact of machine learning,
+rather than technical details of algorithms.
+
+The following books provide excellent overviews of the various ways in which machine
+learning algorithms can have harmful effects:
+
+- `Weapons of Math Destruction <https://weaponsofmathdestructionbook.com/>`_ by Cathy O'Neill
```

Comment:
If you check the user guide there's a way to have sphinx generate a references section.

Reply:
Did you mean:
https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html
?

Reply:
I meant our qucikstart page. It has a references section at the bottom.

Reply:
Fairness in machine learning, perhaps? And having things like that all over the place are exactly why you'd want to use BibTeX.

Reply:
These pages usually need to have just references to whatever is relevant to the individual page. That's how scikit-learn does it. I don't think it's a big issue to duplicate a few of them on a few pages. It's not like they'll change anytime soon.

Reply:
I won't be doing it this PR, but... I disagree about copying references. Having them all in BibTex is a huge boon, whether for correcting typos in the reference (they do happen), or for changing citation style. Same as we try to minimise the copying of code.